src/pages/_app.tsx

```diff
@@ -30,10 +31,31 @@ function App({ Component, pageProps }: AppProps) {
           `,
         }}
       />
-      <ThemeProvider theme={theme}>
-        <Component {...pageProps} />
-      </ThemeProvider>
     </>
+    <ThemeProvider theme={theme}>
+      <Helmet>
+        <meta charSet="utf-8" />
+        <title>Your Website Title</title>
```

Comment:
I would like to mention that we have configured title/description here: https://github.com/geniusyield/atlas-docs/blob/main/theme.config.tsx#L16, want to make sure that you are aware of it.

Reply:
Updated